client/scripts/views/pages/validators/substrate/validator_row.ts

```diff
@@ -121,22 +127,24 @@ const ValidatorRow = makeDynamicComponent<IValidatorAttrs, IValidatorState>({
       (!vnode.attrs.waiting
         && m('td.val-own', formatCoin(vnode.attrs.nominated, true))
       ),
-      m('td.val-commission', stakingInfo?.commission || '-'),
+      m('td.val-commission', stakingInfo?.commission || ' '),
+      (!vnode.attrs.waiting
+        && m('td.val-points', vnode.attrs.eraPoints || ' ')
+      ),
       (!vnode.attrs.waiting
-        && m('td.val-points', vnode.attrs.eraPoints || '-')
+        && m('td.val-last-hash', byAuthor[vnode.attrs.stash] || ' ')
       ),
+      m('td.val-to-be-elected', vnode.attrs.toBeElected
+        ? m(Icon, { name: Icons.ARROW_RIGHT_CIRCLE, size: 'sm' })
+        : ''),
       (vnode.attrs.waiting
```

Comment:
It looks like you replaced the `toBeElected` checks for the `ImOnline` stat, but you didn't carry over the waiting check (instead its being applied to ImOnline which it shouldn't)

Reply:
I have updated `toBeElected ` to Icon, both exist.